### PR TITLE
HLM-575 | remove manage, certification roles and code refactoring in vc-certify-consumer

### PR DESCRIPTION
**Pull request by ajay-beehyv**
@prasanna-egov @varadeth  @saiprakash-v

### CODE REVIEW

File: backend/vc-certify-consumer/main.js [hunk 1]
```diff
@@ -36,12 +36,13 @@ const producer = kafka.producer({allowAutoTopicCreation: true});
...
         }
         while (R.pathOr("",["response","data","params","status"], resp) === REGISTRY_FAILED_STATUS && R.pathOr("",["response","data","params","errmsg"], resp).includes(DUPLICATE_MSG));
-        const certificateStatus = R.pathOr("",["params","status"], resp);
+        let certificateStatus = R.pathOr("",["params","status"], resp);
         if(certificateStatus === REGISTRY_SUCCESS_STATUS){
           console.log("Certificate is created successfully");
           console.log("Response : ", resp);
         }else {
-          console.log("Unable to create certificate: ", resp);
+          certificateStatus = REGISTRY_FAILED_STATUS;
```
Comment (varadeth) [inline backend/vc-certify-consumer/main.js:44]: is this change required?
this can be removed

**Review by varadeth — CHANGES_REQUESTED** [review]

File: backend/vc-certify-consumer/main.js (hunk 1 above)
Comment (ajay-beehyv) [reply backend/vc-certify-consumer/main.js:44]: Yeah, we need certificateStatus in post-vc-certify producer.So if it can't read from response, certificateStatus would be null

**Review by varadeth — APPROVED** [review]

**Review by prasanna-egov — APPROVED** [review]
